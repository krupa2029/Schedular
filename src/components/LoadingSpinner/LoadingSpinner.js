import React from "react";
import classes from "../Styles/LoadingSpinner.module.css";

const LoadingSpinner = () => {
  return <div className={classes.spinner}></div>;
};

export default LoadingSpinner;
