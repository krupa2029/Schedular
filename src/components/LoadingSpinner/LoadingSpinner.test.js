import React from 'react';
import renderer from 'react-test-renderer';
import LoadingSpinner from './LoadingSpinner';

test('should render loading spinner', () => {
  const tree = renderer.create(<LoadingSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
