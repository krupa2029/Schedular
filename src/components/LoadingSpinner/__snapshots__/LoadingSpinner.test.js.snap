// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading spinner 1`] = `<div />`;
